### DISCUSSION TITLE
How to correctly configure it for android.

### DISCUSSION THREAD

**Discussion by Hudson-AmalembaL** [osamaqarem/react-native-image-colors#46] [Q&A] [answered]
Hello, 
I have come across this package and i'm trying to implement it in my app but can't seem to make it work. Any help will be appreciated. Thanks.

**Comment by osamaqarem** [comment, marked as answer]
Edit: this answer is only valid for v1, the steps are different for v2+. Please check the steps in the README.

Hello. There are two steps to install it for Android on the README page.
1- `npm add react-native-image-colors` or `yarn add react-native-image-colors`
2-  ​building the app.
Can you provide more info? What are you facing?
